src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1641,13 +1642,27 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                   call mpas_pool_get_array(diagnosticsPool,'activeTracerSurfaceFluxTendency',activeTracerSurfaceFluxTendency)
                   call mpas_pool_get_array(diagnosticsPool,'temperatureShortWaveTendency',temperatureShortWaveTendency)
                   call mpas_pool_get_array(diagnosticsPool,'activeTracerNonLocalTendency',activeTracerNonLocalTendency)
-               !$omp do schedule(runtime) private(k)
+                  call mpas_pool_get_array(diagnosticsPool,'activeTracerHorizontalAdvectionEdgeFlux', &
+                          activeTracerHorizontalAdvectionEdgeFlux)
+                  call mpas_pool_get_array(diagnosticsPool, 'layerThicknessEdge', layerThicknessEdge)
+
+                  !$omp do schedule(runtime) private(k)
+                  do iEdge = 1, nEdges
+                     do k= 1, maxLevelEdgeTop(iEdge)
+                        activeTracerHorizontalAdvectionEdgeFlux(:,k,iEdge) = &
+                          activeTracerHorizontalAdvectionEdgeFlux(:,k,iEdge) / &
+                          layerThicknessEdge(k,iEdge)
```

Comment:
It would be better to not divide by layer thickness.  If this section is removed, then the units of `activeTracerHorizontalAdvectionEdgeFlux` are `C m^3/s` for `h*u*T*dvEdge` (or `PSU m^3/s`).  The final formula for heat transport is `h*u*T*dvEdge*cp*rho`.  Removing this section just means that the `h` is multiplied on a per timestep basis, which is a more precise measure of heat transport, rather multiplying a mean `h` in post-processing.

Reply:
I disagree as I don't think heat transport is the only consideration here.  This code was added for temperature and salinity budgets primarily.  If we remove the division by h, the temperature and salinity budgets will require post processing by dividing by h to close the budget.  So either way post processing is necessary.  My slight preference is for the post processing to be in heat transport, given that the long term solution is to fix the regional capability in the AM and not rely on this output.

Reply:
Another question, could you explain how you got the units?  In the advection code, we assign the cell center tendency to be work_tend, implying units of C m/s.  Work tend is the sum of ```signedFactor*high_order_flux(edge)```. wouldn't this mean high_order_flux is C m^2/s since it is a horizontal line integral, I"m confused why it is m^3.